cmd/wallet-web/src/pages/Store.vue

```diff
@@ -208,6 +220,10 @@ export default {
           this.errors.push(`failed to save credential`);
         });
     },
+    vaultSelected: function (e) {
+      this.selectedVault = e;
+      return this.selectedVault;
```

Comment:
just curious, why you have to return selected vault here which will always be `e`

Reply:
yes removed returned, not required